### PR TITLE
22 netjson route

### LINKED ISSUE #22: Add simple route for sending netjson dummy data
Add a simple compojure route for handling GET requests for dummy netjson data

### PR DESCRIPTION
**Pull request by skykanin**
Added seperate routing for sending netjson networkgraph dummy data through the api using Compojure.
Closes #22

### CODE REVIEW

**Comment by skykanin** [comment]
I think we should probably introduce ring at this point since it has useful rest api abstractions and middleware. https://github.com/ring-clojure/ring https://github.com/ring-clojure/ring-json

File: backend/src/backend/routes/netjson_route.clj [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  [_]
+  {:status  200
+   :headers {"Content-Type" "application/json"}
+   :body    (slurp (io/resource "networkgraph.json"))})
+
+(defn error-handler
+  "HTTP error response"
+  [_]
+  {:status  404
+   :headers {"Content-Type" "application/json"}
```
Comment (jfremstad) [inline backend/src/backend/routes/netjson_route.clj]: Skal 404 sende json-data?

File: backend/src/backend/routes/netjson_route.clj [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+(defn index
+  "Says hello, world"
+  [_]
+  {:status  200
+   :headers {"Content-Type" "text/html"}
+   :body    (str "Hello, World")})
+
+(defn dummy-data
+  "HTTP response for dummy networkgraph data"
+  [_]
```
Comment (YngveMolnes) [inline backend/src/backend/routes/netjson_route.clj]: What does the _ do, is it just the request or is it some clojure magic?

**Comment by jfremstad** [comment]
It just ignores arguements, I would assume.

**Pull request merged**